nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I think it would be helpful to show the before vs. after, rather than just having a test.  The test can be here, hidden, but I think pretty printing the cell contents before vs after is useful.  </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>